backend/spec/features/admin/orders/listing_spec.rb

```diff
@@ -58,6 +58,16 @@
       within("table#listing_orders") { expect(page).not_to have_content("R100") }
     end
 
+    it "should return both complete and incomplete orders when only complete orders is not checked" do
+      Spree::Order.create! email: "incomplete@example.com", completed_at: nil, state: 'cart'
+      click_on 'Filter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.